pkg/proc/native/proc_freebsd.go

```diff
@@ -85,10 +85,12 @@ func Launch(cmd []string, wd string, foreground bool, debugInfoDirs []string, tt
 	dbp.childProcess = true
 	_, _, err = dbp.wait(process.Process.Pid, 0)
 	if err != nil {
+		_ = dbp.Detach(true)
```

Comment:
This (and all other such occurrences) seem like they would be perfectly suited for a `defer` call. I'd rather something like this:

```
defer func() { 
        if err != nil {
                dbp.Detach(true)
        }
}()
```

Reply:
Done